packages/btcindexer/src/models.ts

```diff
@@ -2,6 +2,25 @@ import { Transaction } from "bitcoinjs-lib";
 import { BtcNet, type BlockQueueRecord } from "@gonative-cc/lib/nbtc";
 import type { SuiNet } from "@gonative-cc/lib/nsui";
 
+export interface NetworkConfig {
+	nbtcPkg: string;
+	suiNetwork: SuiNet;
+}
+
+export interface TxOutput {
+	vout: number;
+}
```

Comment:
this is not needed - If there is nothing else attached to interface (method etc...) then there is no point for this type